packages/js/src/plugins/auctionHouseModule/operations/executeSale.ts

```diff
@@ -406,7 +436,7 @@ export const executeSaleBuilder = (
   // Receipt.
   const shouldPrintReceipt =
     (params.printReceipt ?? true) &&
-    Boolean(listing.receiptAddress && bid.receiptAddress);
+    Boolean(listing.receiptAddress && bid.receiptAddress && !isPartialSale);
```

Comment:
Interesting, does printing a purchase receipt not work with partial sales?

Reply:
Yes, the program expects an ExecuteSale instruction and doesn't support ExecutePartialSale. 

https://github.com/metaplex-foundation/metaplex-program-library/blob/master/auction-house/program/src/receipt/mod.rs#L472

As you can see it tries to deserialize the ExecuteSale instruction.

Reply:
Thanks for the update! 🍺